lib/composables/Table.ts

```diff
@@ -116,6 +118,18 @@ export interface TableCellNumber<V = any, R = any> extends TableCellBase {
   onClick?(value: V, record: R): void
 }
 
+export interface TableCellPath extends TableCellBase {
+  type: 'path'
+  items: TableCellPathItem[]
```

Comment:
@brc-dd 
Do you think `items` is good 🤔 Feels too bit generic I think but things like `segments` felt too long too 🫠

Reply:
yeah segment would've been better 😅

Reply:
lol OK, I'll update this to segment. And also `TableCellPathItem` -> `TableCellPathSegment`